CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.5.1 / 2021-01-15
+
+* [IMPROVEMENT] Cache basic authentication results to significantly improve
```

Comment:
Don't we usually call this `[ENHANCEMENT]`?

Reply:
Yes we do :)

Reply:
Also should I write `performances` or `performance`?

Reply:
I'd say `performance` is better.

`performances` sounds like a number of dance performances or something. :-)

Reply:
👯